tests/IResearch/IResearchFilterArrayInTest.cpp

```diff
@@ -81,12 +81,169 @@
 
 using iterator = std::vector<irs::filter::ptr>::const_iterator;
 
-static const VPackBuilder systemDatabaseBuilder = dbArgsBuilder();
-static const VPackSlice systemDatabaseArgs = systemDatabaseBuilder.slice();
+namespace {
+const VPackBuilder systemDatabaseBuilder = dbArgsBuilder();
+const VPackSlice systemDatabaseArgs = systemDatabaseBuilder.slice();
+
+using FilterIterator =
+    decltype(reinterpret_cast<irs::boolean_filter*>(0)->begin());
+
+void checkTermFilter(FilterIterator begin, bool){// 1st filter
+                                                 {irs::by_term expected;
+*expected.mutable_field() = mangleStringIdentity("a.b.c.e.f");
+expected.mutable_options()->term =
+    irs::ViewCast<irs::byte_type>(std::string_view("1"));
+EXPECT_EQ(expected, **begin);
+}  // namespace
```

Comment:
nit: indentation here is somewhat broken.
I guess this is due to the comment at the beginning of the function body.

Reply:
yes, `{ // comment ` formatted bad